unified_planning/model/problem.py

```diff
@@ -226,10 +226,17 @@ def has_fluent(self, name: str) -> bool:
                 return True
         return False
 
-    def add_fluent(self, fluent: 'up.model.fluent.Fluent', *,
-                   default_initial_value: Union['up.model.fnode.FNode', 'up.model.object.Object', bool,
-                                                int, float, Fraction] = None):
+    def add_fluent(self, fluent_or_name: Union['up.model.fluent.Fluent', str], 
+                   typename: 'up.model.types.Type' = None, *,
+                   default_initial_value: Union['up.model.fnode.FNode', 'up.model.object.Object',
+                   bool, int, float, Fraction] = None
```

Comment:
@arbimo I forgot the comma in my last suggestion:
```suggestion
                   bool, int, float, Fraction] = None,
```